go/examples/webhook-server/go.mod

```diff
@@ -0,0 +1,9 @@
+module webhook-server
+
+go 1.18
+
+require (
+	github.com/Truelayer/truelayer-signing/go v0.1.4 // indirect
+	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
```

Comment:
If anyone knows about a better http cache...
[This](https://github.com/gregjones/httpcache) is not actively maintained but it does the job.